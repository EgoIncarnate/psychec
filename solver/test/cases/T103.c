void f() {
    a.x = 1;
    a.y = "foo";

    b.x = "foo";
    b.y = 9.9;
    b.x = "bar";
}

