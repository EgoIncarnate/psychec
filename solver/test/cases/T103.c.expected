struct __T0__ {int x;char * y;} a ;
struct __T1__ {char * x;double y;} b ;
